site/cypress/e2e/Cypress-Feature/test_forum.spec.js

```diff
@@ -45,6 +46,20 @@ const removeThread = (title) => {
     cy.get('.thread-left-cont > .thread-list-item').contains(title).should('not.exist');
 };
 
+const replyDisabled = (title, attachment) => {
+    cy.get('.thread-left-cont > .thread-list-item').contains(title).click();
+    // Reply button should be disabled by default with no text
+    cy.get('[value="Submit Reply to All"]').should('be.disabled');
+
+    // Ensure reply button is not disabled when attachments are added
+    cy.get('#input-file2').attachFile(attachment);
+    cy.get('[value="Submit Reply to All"]').should('not.be.disabled').click();
+
+    // Wait for submission and ensure attachment with no text is visible
+    cy.get('.attachment-btn').should('exist').click();
```

Comment:
You don't have to have the .should('exist') part because that is implied when you get the attachment button. You should only need to assert existence if you are only checking if it exists, or if it exists, and has a few other parameters (like CSS or class, or id, etc)